notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        X, y, train_size=0.8, test_size=0.2, random_state=None
</pre><p>Why not provide a random state? Without it your results will be more difficult to reproduce.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I agree, we can add that too.